lib/fx/adapters/postgres/functions.rb

```diff
@@ -8,7 +8,7 @@ class Postgres
       class Functions
         # The SQL query used by F(x) to retrieve the functions considered
         # dumpable into `db/schema.rb`.
-        FUNCTIONS_WITH_DEFINITIONS_QUERY = <<-EOS.freeze
+        PG10_FUNCTIONS_WITH_DEFINITIONS_QUERY = <<-SQL.squish.freeze
```

Comment:
Thank you for using `SQL` in the heredoc!

Reply:
Nicer syntax highlighting is always cool :)